inc/utility/namespace.php

```diff
@@ -76,7 +76,7 @@ function group_exports( $dir = null ) {
 
 	$files = [];
 	foreach ( scandir( $dir ) as $file ) {
-		if ( in_array( $file, $ignored, true ) ) {
+		if ( in_array( $file, $ignored, true ) || is_dir( $dir . $file ) ) {
```

Comment:
Add the following test to `\UtilityTest::test_group_exports`

`$this->assertNotContains( 'data', $files );`